plugins/discord/back.js

```diff
@@ -29,7 +31,13 @@ module.exports = (win) => {
 					// Add an idle icon to show that the song is paused
 					activityInfo.smallImageKey = "idle";
 					activityInfo.smallImageText = "idle/paused";
+					// Set start the timer so the activity gets cleared after a while if enabled
+					if (activityTimoutEnabled)
+						clearActivity = setTimeout(()=>rpc.clearActivity(), activityTimoutTime||10,000);
```

Comment:
nit: fallback (10,000) is not the same as the default (60,000), not sure if it's expected

Reply:
Yeah I did that on purpose, if you delete the config / don't have the config, it defaults to 10 seconds, could maybe be changed so that people that are upgrading from older versions generate that config.